### DISCUSSION TITLE
GitHub app crashes on android

### DISCUSSION THREAD

**Discussion by sbrugman** [community/community#45959] [Mobile]
GitHub app crashes/takes minutes to load around half of the time on opening repositories with large readmes. Example: ruff. The phone closes other apps like Spotify because of the load. Quite problematic for being productive with the app. Website doesn't have this issue 

Edit: tracked it down further, I occurs when some google services are unavailable (they are blocked on my device)

Device: Google Pixel 

(Sorry if duplicate)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sbrugman ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by Markwilams56** [comment]
Hey

**Reply by Markwilams56** [reply]
Okay

**Comment by taki-on** [comment]
Hi! Thank you for reporting the issue. I will pass the issue to our Android folks!

**Comment by dennybill** [comment]
One potential solution is to check if there are any updates available for the GitHub app or your Google Pixel device. Updating to the latest version of both the app and the operating system may help to resolve any compatibility issues that may be causing the app to crash or load slowly.

Another solution is to try disabling any unnecessary apps or services on your device that may be consuming resources and causing the app to slow down. This may include apps running in the background or services that are using data or processing power.

Finally, you may want to consider using the GitHub website instead of the app to access repositories with large readmes, as you mentioned that the website does not have this issue. This can help you to stay productive and avoid any frustrations that may arise from using the [gb whatsapp download](https://gbwadownload.pro/gb-whatsapp-download/) on your device.

**Reply by sbrugman** [reply]
Is this ChatGPT generated? Not relevant at all!

The issue is with the Github App crashing when trackers are blocked (apparently it doesn't properly handle when trackers are unavailable)

**Comment by eqdark** [comment]
Hi
